code/modules/reagents/reagent_containers/food/condiment.dm

```diff
@@ -267,3 +267,11 @@
 		..()
 		reagents.add_reagent("vinegar", 50)
 
+/obj/item/weapon/reagent_containers/food/condiment/exotic
+	name = "Exotic Bottle"

```

Comment:
This (and the baster below) are actually better off just lower case entirely to follow naming convention